src/data/library.ts

```diff
@@ -151,6 +122,85 @@ class Library {
     });
   };
 
+  // NOTE this is a high-level public API (eposed on ExcalidrawAPI) with
+  // a slight overhead (always restoring library items). For internal use
+  // where merging isn't needed, use `library.setLibrary()` directly.
+  updateLibrary = async ({
+    libraryItems,
+    prompt = false,
+    merge = false,
+    openLibraryMenu = false,
+    handleError = false,
```

Comment:
This can be completely taken care by host, instead we can expose an api to set error message in the editor, we can just throw error from editor and let host handle it

Reply:
we can, wanted to expose the `setError` in a future PR, but we can do it here as well

Reply:
yup lets do that. I am ok with removing `handleError` here and introducing `setError` in a seprate PR as well but lets not push `handleError`

Reply:
removed the `updateScene.handleError` attribute, but will leave exposing the `setError` for a followup PR.